docs/contributing.rst

```diff
@@ -32,7 +32,7 @@ the team and contributors are working on, you can join our public team calls:
 
 - Mondays and Wednesdays at 3pm CET/CEST.
 
-Both calls take place on `Jitsi <https://meet.ethereum.org/solidity>`_.
+Both calls take place on `Jitsi <http://meet.soliditylang.org/>`_.
 
 How to Report Issues
```

Comment:
This text should be condensed to WHERE to do it.

The information on HOW to do it, should be part of the issue template itself because it is easy to get out of sync.

When you get this merged I'll pick this up.

Reply:
In this comment, I'm referring to "How to report issues".

Reply:
Sounds good. I gave you write permissions so you should be able to submit a PR now.